<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bootloader Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/app.ico</normaloff>:/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QLabel" name="outputFileLabel">
      <property name="text">
       <string>Output file</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="inputFileButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="outputFileBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="appVersionEdit"/>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="protocolBox">
      <item>
       <property name="text">
        <string>0x00000001</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="inputFileLabel">
      <property name="text">
       <string>Input file</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="protocolLabel">
      <property name="text">
       <string>Protocol</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="keyEdit"/>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="inputFileBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="appVersionLabel">
      <property name="text">
       <string>App Version (hex uint32)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="productIdEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="productIdLabel">
      <property name="text">
       <string>Product ID (hex uint32)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="keyLabel">
      <property name="text">
       <string>Key (16 bytes hex)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QComboBox" name="decryptFileBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="outputFileButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QPushButton" name="decryptFileButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Decrypted file</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Key File Path</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="KeyFileBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="keySelectButton">
      <property name="text">
       <string>Select Key</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="keyGenerateButton">
      <property name="text">
       <string>Generate New Key</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
